program/squeezedet/image_benchmark.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
```

Comment:
Can you move this into metadata please, i.e. make a command key and define the env variables for it appropriately e.g. [for individual command](https://github.com/dividiti/ck-tensorrt/blob/master/program/tensorrt-test/.cm/meta.json#L155) and [for all commands](https://github.com/dividiti/ck-tensorrt/blob/master/program/tensorrt-test/.cm/meta.json#L168). You wouldn't need separate scripts for Windows and Linux then?

Reply:
Moved to meta, but I can't defines vars for individual command in `run_set_env2`. Var defined here is not passed as program parameter. So I've stayed them global.